src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,55 @@
+package parser;
+
+import org.junit.jupiter.api.*;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JsonParserTest {
+    private Parser parser;
+    Cart nastyaCart = new Cart("nastya-cart");
+    RealItem car = new RealItem();
+    File file;
+
+    @BeforeEach
+    public void setUp() {
+        parser = new JsonParser();
+
+    }
+
+    @Test
+    void writeToFile() {
+        file = new File("src/main/resources/nastya-cart.json");
+
+        car.setName("Audi");
+        car.setPrice(32026.9);
+        car.setWeight(1560);
+        nastyaCart.addRealItem(car);
+        nastyaCart.showItems();
+
+        parser.writeToFile(nastyaCart);
+
+        assertTrue(file.exists());
+        file.delete();
+    }
+
+    @Disabled
+    void readFromFile_NotNull() {
+        assertNotNull(parser.readFromFile(new File("src/main/resources/eugen-cart.json")));
+    }
+
+    @Test
+    void readFromFile_NoSuchFileException() {
```

Comment:
This test is correct, now modify it. Make this test as parameterized with 5 different datasets. Let me know if you have questions